### ISSUE TITLE
[Bug]: Issue with blocks

### ISSUE THREAD

**Issue by Tonowari** [AlaharranHonor/SWEquestrianM#553]
### What happened?

When breaking a block, and then trying to place a new block in that area, the new block simply turns into the one that was previously placed. Video example - > https://imgur.com/a/FmUGl3t

I asked support for my server's hosting service and they redirected me to the mod author, I asked in the swem server and was redirected here, so that's what I'm doing! :D

How the bug happens -
- Place a block down 
- Break the old block 
- Place a new block 
- Break that block 
- Try to place a new one, **bug happens here**

I found this issue from having tack boxes next to stalls. A player would break it, take it to bind it to the horse, and try to place it again, and they wouldn't open. Break it to try to place a chest down instead, chest turns into the tackbox now. 

This doesn't happen with every block. I've noticed it doesn't *seem* to happen with two vanilla blocks. I've tried multiple different vanilla blocks, and I don't get this bug. But if I break a vanilla block, and then try to place a swem block in that same spot (or vice versa) the bug does occur. 

### What should happen?

The new block should be able to get placed in the old block's spot with no issue. 

### Mods

![ddgdgd](https://user-images.githubusercontent.com/125519918/219227513-3785313d-1714-4b69-b12d-5d876c1dbaef.PNG)

### SWEM Version

SWEM 1.16.5-0.5.3.0

### Singleplayer or Multiplayer

- [x] Singleplayer
- [x] Multiplayer
- [ ] Other (Please add a comment to the issue where this is happening)

### OS

Windows 

### Relevant log output

```shell
[22:02:38] [Server thread/ERROR]: Exception caught during firing event: null
java.lang.NullPointerException
    at com.alaharranhonor.swem.tileentity.LockerTE.getCapability(LockerTE.java:182)
    at net.minecraftforge.common.capabilities.ICapabilityProvider.getCapability(ICapabilityProvider.java:34)
    at dicemc.money.event.EventHandler.onShopBreak(EventHandler.java:137)
```

**Comment by Tonowari** [comment]
- Place a block down 
- Break the old block 
- Place a new block 
- Break that block 
- Try to place a new one, **bug happens here**

I found this issue from having tack boxes next to stalls. A player would break it, take it to bind it to the horse, and try to place it again, and they wouldn't open. Break it to try to place a chest down instead, chest turns into the tackbox now. 

This doesn't happen with every block. I've noticed it doesn't *seem* to happen with two vanilla blocks. I've tried multiple different vanilla blocks, and I don't get this bug. But if I break a vanilla block, and then try to place a swem block in that same spot (or vice versa) the bug does occur.

**Comment by RileyCasper** [comment]
![image](https://user-images.githubusercontent.com/80489042/219510148-13e45ad8-5251-479b-91a7-b95541637991.png)
This bug happens with just this mod list.

[debug.log](https://github.com/AlaharranHonor/SWEquestrianM/files/10761718/debug.log)
[latest.log](https://github.com/AlaharranHonor/SWEquestrianM/files/10761719/latest.log)

**Comment by MagnusHJensen** [comment]
This has been fixed with: `0.5.3.22`
https://www.curseforge.com/minecraft/mc-mods/swem/files/4401759
